src/Entity/Webhook.php

```diff
@@ -25,8 +30,60 @@ final class Webhook
     ];
 
     #[Assert\NotBlank]
+    #[Groups('write')]
     public array $links = [
         'sender' => '',
         'get_submission_url' => '',
     ];
+
+    /**
+     * Internal key used in callback.
+     */
+    public string $remoteApiKey;
+
+    public function getData(): array
+    {
+        return $this->data;
+    }
+
+    public function setData(array $data): self
+    {
+        $this->data = $data;
+
+        return $this;
+    }
+
+    public function getLinks(): array
+    {
+        return $this->links;
+    }
+
+    public function setLinks(array $links): self
+    {
+        $this->links = $links;
+
+        return $this;
+    }
+
+    public function getRemoteApiKey(): string
+    {
+        return $this->remoteApiKey;
+    }
+
+    public function setRemoteApiKey(string $remoteApiKey): self
+    {
+        $this->remoteApiKey = $remoteApiKey;
+
+        return $this;
+    }
+
+    public function getSubmissionURL(): string
+    {
+        $url = $this->links['get_submission_url'];
+
+        // Hack to use when during local testing between containers.
+        // $url = str_replace('http://default', 'http://selvbetjening-nginx-1.frontend:8080', $url);
```

Comment:
Remove.
This can be handled by passing uri to drush in selvbetjening when running queue:

`itkdev-docker-compose drush --uri=http://selvbetjening-nginx-1.frontend:8080 advancedqueue:queue:process os2forms_api_request_handler`

Reply:
:-) removed